docs/source/apidoc/simulation.rst

```diff
@@ -10,10 +10,10 @@ QutipEmulator
 
 :class:`QutipEmulator` is the class to simulate :class:`SequenceSamples`, that are samples of a :class:`Sequence`.
 It is possible to simulate directly a :class:`Sequence` object by using the class method 
-`SequenceSamples.from_sequence`. Since version 0.14.0, the :class:`Simulation` class is deprecated 
-for :class:`QutipEmulator`.
+``SequenceSamples.from_sequence``. Since version 0.14.0, the :class:`Simulation` class is deprecated 
+in favour of :class:`QutipEmulator`.
 
-.. automodule:: pulser_simulation.simulation
+.. autoclass:: pulser_simulation.simulation.QutipEmulator
```

Comment:
So you don't want `Simulation` to show up at all? I think it might be a good idea to still show it, since people will be looking for it.
Add  an `autoclass` for it but without the `:members:`, just so that people are redirected to `QutipEmulator` from the note you just added.

Reply:
Also, though it doesn't show up anymore since you're not using `automodule`, it's a good idea to update the docstring a the top of the `pulser_simulation.simulation` module anyway.